apps/web/pages/api/cron/scheduleReminders.ts

```diff
@@ -0,0 +1,74 @@
+/* Schedule any attendee reminder that falls within 7 days for SMS 
+and 72 hours for email */
```

Comment:
This is the endpoint for the cron job to schedule any reminders that fall within the 7 day window for SMS